app/src/main/java/com/jerboa/Utils.kt

```diff
@@ -832,9 +832,14 @@ enum class ThemeMode(val mode: Int) {
 
 enum class ThemeColor {
     Dynamic,
+    Beach,
+    Blue,
```

Comment:
Needs a merge from main, and you'll see that this enum has i18n strings now.

Reply:
Could you add i18n strings to this, just liken `ThemeMode` has above it?

Reply:
Okay, I've modified them to use i18n strings and added the English translations in the strings.xml file